### PR TITLE
Fixed many sourceURL errors.

### PR DESCRIPTION
**Pull request by kgudger**
Fixed most of the sourceURL errors.
Addresses #103

### CODE REVIEW

File: artwork-data/artworks-overrides.json [hunk 1]
```diff
@@ -2665,7 +2665,7 @@
       "country": "United States",
       "postalCode": "95113",
       "state": "CA",
-      "sourceURL": "https://www.powwowsanjose.com/"
+      "sourceURL": "https://www.sanjoseca.gov/home/showdocument?id=32425"
```
Comment (ychoy) [inline artwork-data/artworks-overrides.json:2668]: Please revert these changes. PowWowSanJose.com murals and public art data are not the same as the City of San Jose public art data. These should still be pointing to PowWowSanJose.com

**Review by ychoy — CHANGES_REQUESTED** [review]
Thanks for updating these links!

For the public art data scraped from the City of San Jose, the new URLs are good to go. 

Please make changes so the PowWowSanJose.com murals do not point to the City of San Jose's website. They should remain the same, pointing to the website PowWowSanJose.com.

File: artwork-data/artworks-overrides.json (hunk 1 above)
Comment (kgudger) [reply artwork-data/artworks-overrides.json:2668]: The only powwowsanjose links that got changed are dead links, so I'm a little confused. Are you suggesting that all dead powwowsanjose links should just be changed to www.powwowsanjose.com ? Or just leave them out (change to blank)? Thanks.

Comment (ychoy) [reply artwork-data/artworks-overrides.json:2668]: These links should not be pointing to the City of San Jose website. That's not where I got my data from. Also the City of San Jose links to public art that is not the same as the murals that are part of the Pow Wow San Jose festival. 

It looks like all the mural location information is now being posted to https://www.powwowsanjose.com/mural-locations

File: artwork-data/artworks-overrides.json [hunk 2]
```diff
@@ -778,7 +778,7 @@
       "country": "United States",
       "postalCode": "95112",
       "state": "CA",
-      "sourceURL": "https://www.powwowsanjose.com/"
+      "sourceURL": "https://www.sanjoseca.gov/home/showdocument?id=32399"
```
Comment (ychoy) [inline artwork-data/artworks-overrides.json:781]: Change this link to https://www.powwowsanjose.com/mural-locations

File: artwork-data/artworks-overrides.json [hunk 3]
```diff
@@ -2576,7 +2576,7 @@
       "country": "United States",
       "postalCode": "95113",
       "state": "CA",
-      "sourceURL": "https://www.powwowsanjose.com/"
+      "sourceURL": "https://www.sanjoseca.gov/home/showdocument?id=32425"
```
Comment (ychoy) [inline artwork-data/artworks-overrides.json:2579]: Change this link to https://www.powwowsanjose.com/mural-locations

Comment (kgudger) [reply artwork-data/artworks-overrides.json:2579]: Am I to understand that all dead powwowsanjose links should be changed to:
https://www.powwowsanjose.com/mural-locations
If so I can do that.

Comment (ychoy) [reply artwork-data/artworks-overrides.json:2579]: Please do, thank you!

Comment (kgudger) [reply artwork-data/artworks-overrides.json]: OK, that should be done in my latest push.

**Review by ychoy — APPROVED** [review]
